mkdocs.yml

```diff
@@ -230,5 +230,7 @@ nav:
           - Nautobot Plugin Nornir: "https://docs.nautobot.com/projects/plugin-nornir/en/latest/"
           - Nautobot Golden Configuration: "https://docs.nautobot.com/projects/golden-config/"
           - Nautobot Single Source of Truth: "https://nautobot-plugin-ssot.readthedocs.io/en/latest/"
+          - Nautobot Device LifeCycle Management: "https://docs.nautobot.com/projects/device-lifecycle/en/latest/"
+          - Nautobot Device Onboarding: "https://docs.nautobot.com/projects/device-onboarding/en/latest/"
```

Comment:
What's the ordering of entries in this section? Should we impose something?

Reply:
The ordering is exactly as it is listed in this doc.  I personally prefer to alphabetize them, but wasn't sure if a separate PR was warranted to do so.  You good if I do so here?

![Screen Shot 2023-03-17 at 12 31 49 PM](https://user-images.githubusercontent.com/536467/226000345-f310f7fa-99ad-4de3-aa71-c6a7bcd34727.png)

Reply:
Original order I think was first-to-get-shiny-new-docs :) Alphabetizing it (based on the second word?) makes a lot of sense.